custodia/custodia

```diff
@@ -44,16 +47,24 @@ def attach_store(typename, plugins, stores):
 CONFIG_SPECIALS = ['authenticators', 'authorizers', 'consumers', 'stores']
 
 def parse_config(cfgfile):
-    parser = RawConfigParser()
+    parser = SafeConfigParser(interpolation=ExtendedInterpolation())
     parser.optionxform = str
-    files = parser.read(cfgfile)
-    if len(files) == 0:
-        raise IOError("Failed to read config file")
+
+    with open(cfgfile) as f:
+        parser.readfp(f)
 
     config = dict()
     for s in CONFIG_SPECIALS:
         config[s] = dict()
+
+    # add env
+    parser['ENV'] = {
+        k: v.replace('$', '$$') for k, v in os.environ.items()
+        if not set(v).intersection('\r\n\x00')}
+
     for s in parser.sections():
+        if s == 'ENV':
```

Comment:
Forgot to ask why this ?
What is the "ENV" section now ? The example shows only ENV prefixed variables, can you add some doc on how we are supposed to use these ENV substitutions ?